vsts/build-test-nuget.yml

```diff
@@ -452,6 +452,66 @@ jobs:
           testRunTitle: "Linux Tests ($(FRAMEWORK)) (Attempt $(System.JobAttempt))"
         condition: always()
 
+
+  ### Upload the unsigned packages to E4K feed ###
+  - job: Upload_to_E4K_Feed
+    displayName: Upload Nuget Packages to E4K Feed
+    timeoutInMinutes: 15
```

Comment:
Have we tested this operation to see how long it takes on average? I don't have a gauge on how relative 15 minutes takes in relation to uploading the packages.

Reply:
on average it takes ~2-3 minutes. This timeout value is similar to the job where we upload to partner signing and release pipeline

Reply:
The timeout value doesn't need to be too restrictive here. 15 minutes is probably fine.